packages/server/src/integrations/tests/utils/postgres.ts

```diff
@@ -15,7 +16,9 @@ export async function getDsConfig(): Promise<Datasource> {
         .withExposedPorts(5432)
         .withEnv("POSTGRES_PASSWORD", "password")
         .withWaitStrategy(
-          Wait.forLogMessage("database system is ready to accept connections")
+          Wait.forLogMessage(
+            "PostgreSQL init process complete; ready for start up."
+          ).withStartupTimeout(new Duration(2, TemporalUnit.SECONDS))
```

Comment:
Could we not instead do a proper healthcheck here? Hit the Postgres port, run a simple query, assert it worked? Very much not a fan of time-based waiting.

Reply:
Same - I've been trying to work out how to do a proper query healthcheck, its not particularly simple, but might just have to go for it. Annoyingly the database has to restart to be fully init'd the first time - so the started message is only correct the second time - which we can't watch for.

Reply:
Switched to `@testcontainers/postgres` - their own prebuilt testcontainer for it!